tests/e2e/common.go

```diff
@@ -425,7 +426,7 @@ func (td *OsmTestData) InstallOSM(instOpts InstallOSMOpts) error {
 		return errors.Wrap(err, "failed to run osm install")
 	}
 
-	return td.WaitForPodsRunningReady(td.osmNamespace, 90*time.Second, 1)
+	return nil
```

Comment:
With this we are not checking if OSM actually came up. I'd be good to have the test fail here if OSM does not come up (ie. ErrPullImage) instead of later when we expect the applications or traffic to happen, which will be misleading when debugging.

Also not saying current implementation is best (as, if there are other containers in the NS, this check doesn't really work), but it did work in most circumstances.

Reply:
We are checking that the osm-controller pod is running, but it's now checked by Helm during `osm install` before that command exits, making this wait in the framework redundant. Any extra wait calls for pods in the osm-system namespace in the individual tests are also made redundant by this change.